scripts/ci-entrypoint.sh

```diff
@@ -103,13 +103,10 @@ setup() {
     export WORKER_MACHINE_COUNT="${WORKER_MACHINE_COUNT:-2}"
     export EXP_CLUSTER_RESOURCE_SET="true"
 
-    # this requires k8s 1.22+
     if [[ -n "${TEST_WINDOWS:-}" ]]; then
         export WINDOWS_WORKER_MACHINE_COUNT="${WINDOWS_WORKER_MACHINE_COUNT:-2}"
         if [[ -n "${K8S_FEATURE_GATES:-}" ]]; then
```

Comment:
I think we can remove the entire if block now. I don't think "if this environment variable is non-empty then set the environment variable value to itself" is doing anything useful.

Reply:
removed